### PR TITLE
Updates based on PR comments

### PR DESCRIPTION
**Pull request by sarahlensing**

### CODE REVIEW

File: app/src/main/kotlin/com/mapzen/erasermap/view/MultiModalDirectionListAdapter.kt [hunk 1]
```diff
@@ -280,26 +262,30 @@ class MultiModalDirectionListAdapter(val context: Context, val instructionGroupe
...
+  }
+
+  fun setTransitRowTextViewsAndIcons(holder: TransitViewHolder, instructionGroup: InstructionGroup,
+      instruction: Instruction) {
+    holder.startingStationName.text = instructionGroup.firstStationName(context, instruction)
+    holder.travelTypeIcon.setImageResource(multiModalHelper.getTransitIcon(
+        instruction.getTravelType()))
+    holder.instructionText.text = instructionGroup.transitInstructionSpannable(instruction)
+    holder.distanceTimeText.text = instructionGroup.numberOfStops(context, instruction)
+    holder.timeView.timeInMinutes = instructionGroup.totalTime / 60
```
Comment (msmollin) [inline app/src/main/kotlin/com/mapzen/erasermap/view/MultiModalDirectionListAdapter.kt]: Use constant here for number of time in seconds?

File: app/src/main/kotlin/com/mapzen/erasermap/view/MultiModalDirectionListAdapter.kt [hunk 2]
```diff
@@ -211,7 +213,7 @@ class MultiModalDirectionListAdapter(val context: Context, val instructionGroupe
     }
     holder.instructionsContainer.removeAllViews()
     if (listItem.expanded) {
-      for (i in 0..instructionGroup.instructions.size - 1) {
+      for (i in instructionGroup.instructions.indices) {
```
Comment (msmollin) [inline app/src/main/kotlin/com/mapzen/erasermap/view/MultiModalDirectionListAdapter.kt]: I was taking a look through the Kotlin reference last night and there appears to be the concept of enumerating items in a collection using a for loop (https://kotlinlang.org/docs/reference/control-flow.html#for-loops)

Would that be of any use in these scenarios? It seems like you're just pulling out the instructions.

File: app/src/main/kotlin/com/mapzen/erasermap/view/MultiModalDirectionListAdapter.kt (hunk 1 above)
Comment (sarahlensing) [inline app/src/main/kotlin/com/mapzen/erasermap/view/MultiModalDirectionListAdapter.kt]: Good catch, updated.

File: app/src/main/kotlin/com/mapzen/erasermap/view/MultiModalDirectionListAdapter.kt (hunk 2 above)
Comment (sarahlensing) [inline app/src/main/kotlin/com/mapzen/erasermap/view/MultiModalDirectionListAdapter.kt]: Ah yes, this is great. Updated to use this syntax.

**Comment by msmollin** [comment]
Merged!

**Pull request merged**